src/pyedb/dotnet/edb_core/stackup.py

```diff
@@ -2107,11 +2107,6 @@ def _import_xml(self, file_path):
         for i in list(lc_import.Layers(self._pedb.edb_api.cell.layer_type_set.AllLayerSet)):
             dumy_layers[i.GetName()] = i.Clone()
 
-        for name in self.layers.keys():
-            if not name in dumy_layers:
-                logger.error("{} doesn't exist in xml".format(name))
-                return False
-
```

Comment:
Would it make sense to log a warning instead of removing the log of an error and return value ?

Reply:
I think that it should check the elevation of the layers, aka the thickness, and the number of them. I changed the code in order to check based on this. @svandenb-dev please review.